core-web/libs/sdk/client/src/lib/client/sdk-js-client.ts

```diff
@@ -19,17 +19,15 @@ export interface ClientConfig {
     /**
      * The id of the site you want to interact with.
      *
-     * @description to get the site id, go to the site you want to interact with and copy the id from the History tab
-     *
+     * @description To get the site id, go to the site you want to interact with and copy the id from the History tab.
```

Comment:
Add a Link to the documentation of the site management.

Also we need to mention that if not pass we will default to the default site.